openfisca_france/model/prestations/minima_sociaux/ppa.py

```diff
@@ -26,6 +26,10 @@ class ppa_eligibilite_etudiants(Variable):
     value_type = bool
     entity = Famille
     label = u"Eligibilité à la PPA (condition sur tout le trimestre)"
+    reference = [
+        u"Article L842-2 du Code de la Sécurité Sociale",
```

Comment:
```suggestion
        # Article L842-2 du Code de la Sécurité Sociale
```

Sinon, sur legislation explorer, on ajoute de liens morts :

<img width="808" alt="capture d ecran 2018-12-28 a 15 38 42" src="https://user-images.githubusercontent.com/329236/50518590-b21bc200-0ab6-11e9-99f4-2a3a41cdcd3f.png">

Reply:
Est-qu'un fix peut être appliqué au legislation explorer ?

Reply:
Ce n'est pas un bug, donc on ne peux proprement parler de « fix », mais de fonctionnalité.

Oui, on peut ajouter cette fonctionnalité, si l'on prend en compte la structure de données à utiliser : une `list` est une collection d'objects à accès aléatoire.

Pour ce que l'on veut, il faudrait utiliser un `dictionary` ou dictionary-like (a hashmap quoi).

Reply:
On ne peut pas, de façon déterministe, construire une paire clé-valeur à partir d'une liste de strings.

Reply:
(dans tous les cas, note que c'est une suggestion, pas un request changes)